pokerole_irl/worlds_app/models.py

```diff
@@ -1,34 +1,52 @@
 from django.db import models
 from django.contrib.auth.models import User
+from django.utils.text import slugify
+from django.urls import reverse
+
 
 # Create your models here.
 class World(models.Model):
     members = models.ManyToManyField(User, through='WorldMember')
     name = models.CharField(max_length=100)
     description = models.TextField(default="")
-
+    world_slug = models.SlugField(unique=True)
 
     def __str__(self):
         return self.name
 
+    def save(self, *args, **kwargs):
+        if not self.world_slug:
+            self.world_slug = self.world_slug or slugify(self.name)
```

Comment:
Hm, why do we set self.world_slug to self.world_slug here, when we know it's falsy? Surely it's fine to just do

```suggestion
        self.world_slug = slugify(self.name)
```